packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1370,11 +1372,38 @@ abstract class RenderBox extends RenderObject {
       return true;
     }());
     if (shouldCache) {
+      Map<String, String> _debugTimelineIntrinsicsArguments = timelineArgumentsIndicatingLandmarkEvent;
+      assert(() {
+        if (debugProfileLayoutsEnabled) {
+          _debugTimelineIntrinsicsArguments = toDiagnosticsNode().toTimelineArguments();
+        } else {
+          _debugTimelineIntrinsicsArguments = Map<String, String>.of(_debugTimelineIntrinsicsArguments);
+        }
+        _debugTimelineIntrinsicsArguments['intrinsics dimension'] = describeEnum(dimension);
+        _debugTimelineIntrinsicsArguments['intrinsics argument'] = '$argument';
+        return true;
+      }());
+      if (!kReleaseMode) {
+        if (debugProfileLayoutsEnabled || _debugIntrinsicsDepth == 0) {
```

Comment:
Wondering if it is really useful to show the full path when this is enabled? It will make it look like as if every step is just contributing a tiny amount and it will also make it harder to spot this in traces.

Reply:
Showing the full path would be useful if one where to optimize the intrinsics calculations itself, but that seems rare?

Reply:
Mostly I was trying to show the depth of the tree being measured, just like we show the depth of the tree being built/laid out/painted.

Reply:
Makes sense. We can always turn this off in the future if it turns out to not be helpful.